model/validation.go

```diff
@@ -838,6 +838,55 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs
 				parentShutdownDuration.Seconds(), drainDuration.Seconds()))
 	}
 
+	if drainDuration > (time.Hour * 1) {
```

Comment:
General comment: can you use explicit `return err` throughout instead of named parameters? There is only one return value `errs` so I don't think the named part adds much.